config/config.js

```diff
@@ -0,0 +1,30 @@
+import env from "../config/env.generated.json";
+
+let config = {
+  userLoggedInStatus: false,
+  username: ``
+};
+
+export default {
+  logInUrl: {
+    get: function(redirectUrl = env.ORIGIN) {
+      return `${env.PULSE_API}/login?original_url=${encodeURIComponent(redirectUrl)}`;
```

Comment:
Hm, why is redirectUrl not simply `window.location.toString()` here, so that users always get back to whatever page they're on right now?